### PR TITLE
Add recipe for pytriangle

### PR DESCRIPTION
**Pull request by gregvonkuster**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pytriangle/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+test:
+  # Python imports
+  imports:
+    - triangle
+
+requirements:
+  build:
+    - python
+
```
Comment (bgruening) [inline recipes/pytriangle/meta.yaml]: Can you remove this empty line here?

File: recipes/pytriangle/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package:
+  name: pytriangle
+  version: "1.0.0"
+
+about:
+  home: 'https://github.com/pletzer/pytriangle'
+  summary: 'A python interface to the 2D triangulation program TRIANGLE (https://www.cs.cmu.edu/%7Equake/triangle.html)'
```
Comment (bgruening) [inline recipes/pytriangle/meta.yaml]: The URL can be removed from the description I think.

**Comment by bgruening** [comment]
Looks good to me. Feel free to merge after the style changes.

**Comment by gregvonkuster** [comment]
Thanks @bgruening - fixes here:https://github.com/bioconda/bioconda-recipes/pull/2098/commits/d622dc1d9941de473b795f69979c214daf85d674

**Pull request merged**